man/mkosi.1

```diff
@@ -416,7 +416,7 @@ the image.
 Takes a comma separated list of packages.
 This option may be used multiple times in which case the specified
 package list is combined.
-Packaged specified this way will be installed both in the development
+Packages specified this way will be installed both in the development
```

Comment:
This needs to fixed in `mkosi.md` instead of here, since this file is generated from it.